.gitattributes

```diff
@@ -10,6 +10,8 @@
 *.RTF	diff=astextplain
 
 *.ps1 eol=crlf
+*.sh eol=lf
+*.config eol=lf

```

Comment:
The only .config files in this repository are `Nuget.config` and `app.config` (to use the server GC), so I don't think this should be added.

Actually, I think it's best to not have this file at all. As described [here](https://stackoverflow.com/questions/57030698/do-i-really-need-to-specify-all-binary-files-in-gitattributes), git is pretty good at binary file detection. In this repository there's one .pdf file and a few .png files, all of which contain a 0x00 byte early in the file.

Alternatively, I'm fine with having solely `*.sh`, for convenience when running Linux scripts on Windows. The downside of having all the other files is that VS pops up a dialog asking to convert to CRLF when opening the file. If vscode appears to auto-convert to LF on Windows for some reason, it should be [configured](https://stackoverflow.com/questions/60560549/vscode-changing-crfl-to-lf-on-file-save-and-git-records-a-change-even-if-there) not to do so.

Reply:
I've pushed a commit to clean up this file as proposed.